nuxeo-core/nuxeo-core-redis/src/main/java/org/nuxeo/ecm/core/redis/contribs/RedisKeyValueStore.java

```diff
@@ -128,6 +131,59 @@ public void put(String key, byte[] value, long ttl) {
         return redisExecutor.execute(jedis -> jedis.get(getBytes(namespace + key)));
     }
 
+    /**
+     * @since 9.10
+     */
+    @Override
+    public Map<String, byte[]> get(Set<String> keys) {
+        Objects.requireNonNull(keys);
+        Map<String, byte[]> kvMap = new HashMap<>(keys.size());
```

Comment:
Could you rename the `kvMap` in this and other files too?

Reply:
The patch has been revised to also include these changes.